assets/src/domain/eventEditor/ui/tickets/ticketsList/tableView/ticketsListTableHeader.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
```

Comment:
`registrations` column has been removed as in datetime case

Reply:
`reserved` one should stay intact ?

Reply:
no i think Josh asked us to remove the reserved counts

Reply:
ok, then `registrations` should remain intact

Reply:
in v1 I see reservation count, not reserved count:
![image](https://user-images.githubusercontent.com/1477453/76110969-21107700-5fe8-11ea-9295-1cf26ee88bec.png)

Reply:
>  I see reservation count, not reserved count:

???

do you mean `registrationCount` ?

there was a "Reserved" column and a "Registrations" column in EDTR v1

but we do not want to display the reserved column anymore